gtda/diagrams/representations.py

```diff
@@ -179,51 +179,51 @@ def plot(self, Xt, sample=0, homology_dimensions=None):
             for dim in homology_dimensions:
                 if dim not in self.homology_dimensions_:
                     raise ValueError(
-                        f'All homology dimensions must be in '
-                        f'self.homology_dimensions_ which is '
-                        f'{self.homology_dimensions_}. {dim} is not.')
+                        f"All homology dimensions must be in "
```

Comment:
I'm starting to use this type of quotation marks consistently in sentence-like strings as they are safe when apostrophes have to be used inside as well.